packages/aragon-wrapper/src/index.js

```diff
@@ -242,7 +242,7 @@ export default class Aragon {
     const REORG_SAFETY_BLOCK_AGE = 100
 
     const currentBlock = await this.web3.eth.getBlockNumber()
-    const cacheBlockHeight = currentBlock - REORG_SAFETY_BLOCK_AGE
+    const cacheBlockHeight = Math.min(currentBlock - REORG_SAFETY_BLOCK_AGE, 0) // clamp to 0 for safety
```

Comment:
```suggestion
    const cacheBlockHeight = Math.max(currentBlock - REORG_SAFETY_BLOCK_AGE, 0) // clamp to 0 for safety
```

We probably want to use `max` as min will always take `0` when `0` and `7801359` are passed in.

Reply:
Good catch 😄